datanode/sqlstore/migrations/0046_additional_referral_set_stats.sql

```diff
@@ -5,10 +5,16 @@ alter table referral_set_stats
 alter table referral_set_stats
     add column rewards_factor_multiplier text not null default '0';
 
+alter table referral_set_stats
```

Comment:
is this migration on mainnet already, or not? (I suspect it isn't, but still thought I'd comment)

Reply:
It's not.